mayhem/fuzz-stegano.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/python3
```

Comment:
Are these valid exceptions or simply fuzzing errors?

Reply:
These are either expected exceptions, or exceptions in underlying libraries most notably PIL.